fgl.cabal

```diff
@@ -67,6 +68,7 @@ library {
     build-depends:    base < 5
                     , transformers
                     , array
+                    , vector
```

Comment:
Pretty much; you can also see what the Changelog says (if there is one).

It might suffice to just say `>= 0.10 && < 0.12` as 0.10 claims to work with base-4 (which was first introduced with GHC 6.10 in November 2008); I think that will do (just put a new lower-bound on base whilst you're at it to make it more obvious).